presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -179,10 +179,9 @@ void PrestoServer::run() {
   protocol::registerHiveConnectors();
   protocol::registerTpchConnector();
 
-  auto executor = std::make_shared<folly::IOThreadPoolExecutor>(
+  auto ioExecutor = std::make_shared<folly::IOThreadPoolExecutor>(
```

Comment:
Move the creation right above where it is used? Also can you make this a member of presto server? such as the other executors like cacheExecutor_? And having a server startup log such as
```

  PRESTO_STARTUP_LOG(INFO) << "Driver CPU executor has "
                           << driverCPUExecutor()->numThreads() << " threads.";
```